cmd/erasure-readfile.go

```diff
@@ -125,11 +126,15 @@ func parallelRead(volume, path string, readDisks []StorageAPI, orderedDisks []St
 		go func(index int) {
 			defer wg.Done()
 
-			// Verify bit rot for the file on this disk.
-			if !bitRotVerify(index) {
-				// So that we don't read from this disk for the next block.
-				orderedDisks[index] = nil
-				return
+			// evaluate if we need to perform bit-rot checking
+			needBitRotVerification := true
+			if brVerifiers[index].isVerified {
+				needBitRotVerification = false
+				// if file has bit-rot, do not reuse disk
+				if !brVerifiers[index].hasNoBitRot {
```

Comment:
Consider renaming hasNoBitRot to simply hasBitRot?
Then we can simply do `if brVerifiers[index].hasBitRot` here instead of 'double' negation.
Also down in line 164 we don't need need to invert
(and the default false value would, for most cases, better reflect the real-world status)